cloudflare-mixin/README.md

```diff
@@ -0,0 +1,89 @@
+# Cloudflare Mixin
+The Cloudflare mixin is a set of configurable Grafana dashboards and alerts. 
+
+The metrics analyzed here are a part of `Cloudflare Analytics - account and zone analytics` found [here](https://developers.cloudflare.com/analytics/account-and-zone-analytics/). Pool and worker metrics will not appear until your Cloudflare instance has been configured to utilize them.
+
+The Cloudflare mixin contains the following dashboards:
+
+- Cloudflare zone overview
+- Cloudflare GeoMap overview
```

Comment:
shoud GeoMap be lowercase?

Reply:
Eh, thought about it but didn't think it looked very good. I'm gonna leave as is and see what Grafana thinks.